internal/controller/utils.go

```diff
@@ -71,21 +71,21 @@ func verifyObject(spec acpv1.AzureAppConfigurationProviderSpec) error {
 
 		// Check if feature flag label filters are valid
 		for i := range spec.FeatureFlag.Selectors {
-			if spec.FeatureFlag.Selectors[i].LabelFilter != nil && hasNonEscapedValueInLabel(*spec.FeatureFlag.Selectors[i].LabelFilter) {
-				return loader.NewArgumentError("spec.featureFlag.selectors", fmt.Errorf("non-escaped reserved wildcard character '*' and multiple labels separator ',' are not supported in label filters"))
+			err = verifySelectorObject(spec.FeatureFlag.Selectors[i])
```

Comment:
FeatureFlag is verifying input the same way asa Configuration.Selectors. Does this mean FeatureFlag supports Snapshot filter now?

Reply:
Yes, it will filter out all the feature flag type key-values in specified snapshot